### PR TITLE
{AKS} Update unit test check-in pipeline

### PR DESCRIPTION
**Pull request by FumingZhang**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- Add coverage inspection in unit test check-in pipeline, error will be reported when it is lower than the preset value.
- Fix the testsdk changes introduced in [PR #20601](https://github.com/Azure/azure-cli/pull/20601).

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/aks-preview/azcli_aks_live_test/configs/ext_coverage_config.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "totals": {
+        "fail_percent": 59,
```
Comment (andyliuliming) [inline src/aks-preview/azcli_aks_live_test/configs/ext_coverage_config.json:3]: why is this different with cli_coverage_config.json

File: src/aks-preview/azcli_aks_live_test/scripts/test_cli_unit.sh [hunk 2]
```diff
@@ -9,6 +9,8 @@ set -o xtrace
 # check var
 [[ -z "${WIKI_LINK}" ]] && (echo "WIKI_LINK is empty"; exit 1)
 [[ -z "${ACS_BASE_DIR}" ]] && (echo "ACS_BASE_DIR is empty"; exit 1)
+[[ -z "${IGNORE_EXIT_CODE}" ]] && (echo "IGNORE_EXIT_CODE is empty")
```
Comment (andyliuliming) [inline src/aks-preview/azcli_aks_live_test/scripts/test_cli_unit.sh:12]: curious where is this used? in which case we ignore exit code.

Comment (FumingZhang) [reply src/aks-preview/azcli_aks_live_test/scripts/test_cli_unit.sh:12]: Actually this is not used by the check-in pipeline, we always exit with the error code.

This is used in our runner, even some of the test cases may fail, the status of the pod/container after execution should be completed.

File: src/aks-preview/azcli_aks_live_test/configs/ext_coverage_config.json (hunk 1 above)
Comment (FumingZhang) [reply src/aks-preview/azcli_aks_live_test/configs/ext_coverage_config.json:3]: For the current situation, the overall unit test coverage in acs (azure-cli) is merely above 70%. For aks-preview (azure-cli-extensions) the value is 59%+. There is a big gap between the two modules, so I am using two configuration files to control them separately.

**Review by andyliuliming — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**